lightning_server.cc

```diff
@@ -51,14 +50,21 @@ void LightningServer::start() {
     }
 
     // Handle echo response in external handler
-    char response_buffer[MAX_REQ_SIZE];
-    size_t response_size = bytes_read;
-    request_handlers::echoRequestHandler(request_buffer,
-                                         response_buffer,
-                                         response_size);
+    char* response_buffer = nullptr;
+    size_t response_buffer_size  = 0;
+
+    EchoRequestHandler echo_request_handler;
+    echo_request_handler.handleRequest(request_buffer,
+                                       request_buffer_size,
+                                       response_buffer,
+                                       response_buffer_size);
+
+    // TODO: Use-after-free vulnerability if response_buffer is used after
```

Comment:
Can you add an issue for "better memory management"? That'll let us revisit this thorny issue.